api_docs/changelog.md

```diff
@@ -20,6 +20,20 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 206**
+
+* [`GET /events`](/api/get-events): Added a new flag,
+  `topic_wildcard_mentioned`. It will be available in the `flags`
+  field of the `message` and `update_message` events response for
+  topic wildcard mentions. Now, the `wildcard_mentioned` flag will
+  be exclusively used for stream wildcard mentions.
+* [`GET /messages`](/api/get-messages),
+  [`GET /messages/{message_id}`](/api/get-message): Added a new
+  flag, `topic_wildcard_mentioned`. It will be available in the
+  `flags` field of the message object for topic wildcard mentions.
+  Now, the `wildcard_mentioned` flag will be exclusively used for
+  stream wildcard mentions.
```

Comment:
I would probably compress these a bit more rather than duplicating so much text -- referring to the "`flags` field on message objects" ... would probably suffice to just list all 3 endpoints at the start.

Reply:
changelog updated as we are moving forward with a [new approach](https://chat.zulip.org/#narrow/stream/378-api-design/topic/highlight.20topic-wildcard.20mention/near/1656239)